gsa/src/web/components/form/useFormValidation.js

```diff
@@ -110,7 +105,7 @@ const useFormValidation = (
 };
 
 // Some common validation rules
-export const testNonEmptyString = string => string.length > 0 && string !== ' ';
+export const testNonEmptyString = string => string.trim().length > 0;
```

Comment:
Shouldn't the test function be safe against undefined values? In that case it would be better to use something like
```
(string = '') => string.trim().length > 0;

Reply:
Good idea. Thanks!